docs/en/operations/settings/settings.md

```diff
@@ -1397,6 +1397,32 @@ Minimum count of executing same expression before it is get compiled.
 
 Default value: `3`.
 
+## compile_aggregate_expressions {#compile_aggregate_expressions}
+
+Enables or disables JIT-compilation of aggregate functions to native code. Enabling the `compile_aggregate_expressions` setting can improve the performance.
+
+Possible values:
+
+-   0 — Aggregation is done without JIT compilation.
+-   1 — Aggregation is done using JIT compilation.
+
+Default value: `0`.
+
+**See Also** 
+
+-   [min_count_to_compile_aggregate_expression](#min_count_to_compile_aggregate_expression)
+
+## min_count_to_compile_aggregate_expression {#min_count_to_compile_aggregate_expression}
+
+If the [compile_aggregate_expressions](#compile_aggregate_expressions) setting is enabled and there are more then `min_count_to_compile_aggregate_expression` identical aggregate expressions in a `SELECT` query, then they are compiled JIT.
```

Comment:
```suggestion
If the [compile_aggregate_expressions](#compile_aggregate_expressions) setting is enabled and there are more than `min_count_to_compile_aggregate_expression` identical aggregate expressions in a `SELECT` query, then they are JIT-compiled.
```

Reply:
Maybe it would be better to say

> The minimum number of identical aggregate expressions in a `SELECT` query before they are JIT-compiled. Works only if [compile_aggregate_expressions](#compile_aggregate_expressions) setting is enabled.